docs/step_by_step_guide.md

```diff
@@ -0,0 +1,647 @@
+
+
+## Launch zppy
+Run `zppy -c post.<case_name>.cfg`. This will submit a number of jobs. Run `sq` to see what jobs are running.
+
+`zppy` automatically handles dependencies of jobs. E.g., `e3sm_diags` jobs are dependent on `climo` and `ts` jobs, so they wait for those to finish. MPAS Analysis jobs re-use computations, so they are chained.
+
+Most jobs run quickly, though E3SM Diags may take around an hour and MPAS Analysis may take several hours.
+
+`zppy` creates a new directory `<simulations_dir>/<case_name>/post`. Each realization will have a shell script (typically `bash`). This is the actual file that has been submitted to the `batch` system. There will also be a log file `*.o<job ID>` as well as a `*.status` file. The status file indicates the state (WAITING, RUNNING, OK, ERROR). These files can be found in `<simulations_dir>/<case_name>/post/scripts`. Once all the jobs are complete, you can check their status.
+```
+cd <simulations_dir>/<case_name>/post/scripts
+cat *.status # should be a list of "OK"
+grep -v "OK" *.status # lists files without "OK"
+```
+If you re-run `zppy`, it will check the status of tasks and will skip a task if its status is “OK”. As your simulation progresses, you can update the post-processing years in the configuration file and re-run `zppy`. Newly added task will be submitted, while previously completed ones will be skipped.
+
+## Tasks
+If you run `ls <simulations_dir>/<case_name>/post/scripts` you’ll see files like `e3sm_diags_180x360_aave_model_vs_obs_0001-0020.status`. This is one e3sm_diags job. Parts of this file name are explained below:
+| Part of File Name | Meaning |
+| --- | --- |
+| `e3sm_diags` | Task |
+| `180x360_aave` | Grid |
+| `model_vs_obs` | `model_vs_model` or `model_vs_obs` |
+| `0001-0020` | First and last years |
+There is also a corresponding output file. It will have the same name but end with `.o<job ID>` instead of `.status`.
+
+## Output
+The post-processing output is organized hierarchically. Examples:
+- `<e3sm_simulations_dir>/<case_name>/post/atm/180x360_aave/ts/monthly/10yr` has the time series files – one variable per file, in 10 year periods as defined in `<run_scripts_dir>/post.<case_name>.cfg`.  
+- `<e3sm_simulations_dir>/<case_name>/post/atm/180x360_aave/clim/20yr` similarly has climatology files for 20 year periods, as defined in `<run_scripts_dir>/post.<case_name>.cfg``.
+- `<e3sm_simulations_dir>/<case_name>/post/atm/glb/ts/monthly/10yr` has globally averaged files for 10 years periods as defined in `<run_scripts_dir>/post.<case_name>.cfg`.
+
+# Documenting the Model Run
+You should create a Confluence page for your model run in the relevant Confluence space. Use the [Simulation Run Template](https://acme-climate.atlassian.net/wiki/spaces/EWCG/pages/2297299190) as a template. See below for how to fill out this template.
+
+<!-- TODO: where should the Confluence pages be made for v3 (and for each group)? -->
```

Comment:
Is there a newer template that should be used?  What Confluence directory should v3 simulations be documented in?